custom/enikshay/ucr/reports/qa/beneficiary_register.json

```diff
@@ -50,17 +50,6 @@
         ],
         "columns": [
             {
-                "field": "beneficiary_id",
```

Comment:
It's report change do this will not cause rebuild. This column doesn't exist in data source which was proven by tests that I added. This report is not really used.